src/main/port.ts

```diff
@@ -19,6 +26,14 @@ export async function getFreePort(firstPort: number, lastPort: number): Promise<
     try {
       logger.debug("Checking port " + port + " availability ...")
       await checkPort(port)
+      const resp = await request(`http://127.0.0.1:${port}`, {
```

Comment:
doesn't work as intended: connects to the lens-port-checker OK even if the port is already in use on the 'all' interface.